controller/controller.js

```diff
@@ -21,10 +21,25 @@ const getApiArticles = (request, response) => {
     });
 };
 
+// /api/articles/:article_id Endpoints
+const getArticleById = (request, response, next) => {
+    const articleId = request.params["article_id"];
+
+    model.articles.selectArticleByArticleId(articleId).then((selectArticleByArticleIdResult) => {
```

Comment:
You should complete all data manipulation inside the model so that the `.then` block is invoked with just the article you have requested.